lib/board.rb

```diff
@@ -0,0 +1,21 @@
+class Board
+	def initialize(size, cell)
+		@cells = Array.new(size*size) { cell.new }
+	end
+
+	def number_array()
+		cells.map { |cell| cell.player }
```

Comment:
You don't need to have empty arrays after the function name if there's no parameters. `cells.map(&:player)` is also a simplification—basically maps and applies that function onto the items.